package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "suomifi-design-tokens",
-  "version": "3.1.1",
+  "version": "3.2.0",
```

Comment:
Not exactly how we work with other repos that separate PR for increasing the version number for release.
BUT as tokens really rarely gets updates, I think it is fine like this (and from now on also). 
Easier to directly communicate the change and how it affects the version number.